src/core/sock/fd_collection.cpp

```diff
@@ -532,7 +534,9 @@ int fd_collection::del_sockfd(int fd, bool b_cleanup /*=false*/, bool is_for_udp
         // 2. Socket deletion when TCP connection == CLOSED
         if (p_sfd_api->prepare_to_close()) {
```

Comment:
Can a goal be achieved just ignoring TRUE branch by extra condition?

Reply:
OK, done!